pkgs/applications/science/biology/somatic-sniper/default.nix

```diff
@@ -0,0 +1,26 @@
+{stdenv, fetchFromGitHub, cmake, zlib, ncurses}:
+
+stdenv.mkDerivation rec {
+  name = "somatic-sniper-${version}";
+  version = "1.0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "genome";
+    repo = "somatic-sniper";
+    rev = "v${version}";
+    sha256 = "0lk7p9sp6mp50f6w1nppqhr40fcwy1asw06ivw8w8jvvnwaqf987";
+  };
+
+  patches = [ ./somatic-sniper.patch ];
+
+  buildInputs = [ cmake zlib ncurses ];
+
+  meta = with stdenv.lib; {
+    description = "The purpose of this program is to identify single nucleotide positions that are different between tumor and normal (or in theory, any two bam files).";
```

Comment:
Please use a `description` that conforms to https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.